recipes/icloud-mail/webview.js

```diff
@@ -0,0 +1,49 @@
+function _interopRequireDefault(obj) {
+  return obj && obj.__esModule ? obj : { default: obj };
+}
+
+const _path = _interopRequireDefault(require('path'));
+
+module.exports = Ferdium => {
+  const getMessages = root => {
+    // it's either localised "Inbox" name or unread count.
+    const unread = root.querySelector(
+      `.mailbox-list-pane .mailbox-list .mailbox-list-item:first-child p:last-child`,
+    );
+    if (!unread) return;
+    const count = Number.parseInt(unread.textContent) ?? 0;
```

Comment:
instead of `Number.parseInt` please use `Ferdium.safeParseInt`

Reply:
Done! Though I couldn’t test it since I’m on phone.